src/Search/Api/Elasticsearch/Command/BuildIndexCommand.php

```diff
@@ -67,7 +67,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
             try {
                 $delete = $this->client->deleteIndex();
             } catch (Throwable $e) {
-                $logger->debug($e->getMessage(), $e->getTrace());
```

Comment:
> The issue I've seen is on a clean environment where there is no index in the first instance, and an error is displayed saying "no index to delete"

Agreed that this shouldn't cause an error, but other deletion errors are still interesting to see. Therefore some possible ways to make it work are:
- checking if an index esists first, and deleting it only if it exists
- or, always trying to delete it, but logging with `info` instead of `error` in case the exception is exactly what we expect to happen (checking its class or message); other exceptions should still be errors